index.html

```diff
@@ -0,0 +1,136 @@
+        Skills: Skills: HTML 5, CSS 3 (SAAS/SCSS), JS, AD PFOTOSHOP, AD ILLUSTRATOR, AVACODE etc.;
+      </p>
+    </div>
+  </section>
+  <section class="block block_blue">
+    <div class="wrapper">
+      <h2>Expert In</h2>
+      <div class="flexbox">
+        <div class="flexbox_item">
+          <div class="flexbox_icon flexbox_icon_app"></div>
+          <h3>MOBILE APP DESIGN</h3>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sit deserunt sed exercitationem nostrum cum itaque? Voluptatum unde quia quos a molestias exercitationem ducimus consequatur facere reiciendis aliquid. At, cum dignissimos!
+          </p>
+        </div>
+        <div class="flexbox_item">
+          <div class="flexbox_icon flexbox_icon_design"></div>
+          <h3>WEB DESIGN</h3>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, consequuntur alias deserunt, id officia numquam expedita sapiente aliquam eum incidunt dicta commodi sint autem nemo odit soluta animi assumenda hic?
+          </p>
+        </div>
+        <div class="flexbox_item">
+          <div class="flexbox_icon flexbox_icon_web"></div>
+          <h3>WEB DEVELOPMENT</h3>
+          <p>
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Doloribus dolorem tempora, optio, maiores, est voluptas esse distinctio sed sunt quas necessitatibus ipsam nobis fugiat nemo nesciunt deleniti adipisci atque cumque.
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="block">
+    <div class="wrapper">
+      <h2>Latest Works</h2>
+      <div class="flexbox">
+        <div class="flexbox_item">
+          <!-- чтобы сделать картинки ссылками на работы, нужно обернуть .thumb в <a> и убрать свойство cursor;) -->
+          <div class="thumb">
+            <img src="assets/img/im1.png" alt="">
+          </div>
+        </div>
+        <div class="flexbox_item">
+          <div class="thumb">
+            <img src="assets/img/im2.png" alt="">
+          </div>
+        </div>
+        <div class="flexbox_item">
+          <div class="thumb">
+            <img src="assets/img/im3.png" alt="">
+          </div>
+        </div>
+      </div>
+  </section>
+  <footer class="block_footer">
+    <div class="wrapper">
+      <div class="block_footer_flex">
+        <h2>Need a web developer? <span>Ready to work!</span></h2>
+        <div>
+          <a href="#" class="block_footer_link">hire me</a>
+        </div>
+      </div>
+    </div>
+  </footer>
+  <a href="#" class="btn_up"></a>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script type="text/javascript"> 
+    $(function(){ 
```

Comment:
It wasn't necessary, but if u want to do smth like this, please add a own file for script.